t/07-commands.t

```diff
@@ -119,6 +123,31 @@ subtest 'web socket route' => sub {
     $t->finish_ok();
 };
 
+subtest 'data api' => sub {
+    my $job = 'Hallo';
+    $t->get_ok("$base_url/$job/data");
+    $t->status_is(200);
+    $t->content_type_is("application/x-cpio");
+
+    $t->get_ok("$base_url/$job/data/mod1");
+    $t->status_is(200);
+    $t->content_type_is("application/x-cpio");
```

Comment:
I would like to add a check, if the achieve really contains all files.
How do I check make sure, that cpio is available?

Reply:
do you mean if the cpio command line tool is available in the environment that runs the tests? As "cpio" is included by default even in the docker containers "opensuse/tumbleweed" or even "mini/base" (alpine) I would not be concerned about environments missing it but still check that the command exists

Reply:
Probably better:

```
use File::Which;
ok(-x which 'cpio');
```

Reply:
exact that was my question. Thanks a lot!